Ui/Component/Listing/Column/TrackActions.php

```diff
@@ -78,18 +78,6 @@ public function prepareDataSource(array $dataSource)
                         'label' => __('Download package label'),
                         'hidden' => false,
                     ];
-                    $item[$this->getData('name')]['action-download_letter_label'] = [
```

Comment:
Dit is toch ongefrankeerd? Dat hebben we toch in BE?

Reply:
BE heeft alleen pakket